lib/lookup.json

```diff
@@ -574,12 +565,12 @@
     "maintainers": ["einaros", "3rd-Eden", "lpinca"]
   },
   "yargs": {
-    "comment": "Appears to require npm 8 to successfully install.",
+    "comment": "Install from source is currently broken due to TS error",
     "prefix": "v",
     "expectFail": "fips",
     "maintainers": ["bcoe", "addaleax"],
     "head": true,
-    "skip": ["14"]
+    "skip": true
```

Comment:
Did you try to remove `"head": true` ?

Reply:
still fails with removing head. TS shipped a change that breaks the build in a version that the splat updates (I think a a minor)